### PR TITLE
Added tap plugin and linkInContainer to nad pkg

### PR DESCRIPTION
**Pull request by kononovn**

### CODE REVIEW

File: pkg/nad/masterplugin.go [hunk 1]
```diff
@@ -101,6 +101,20 @@ func (plugin *MasterMacVlanPlugin) WithIPAM(ipam *IPAM) *MasterMacVlanPlugin {
 	return plugin
 }
 
+// WithLinkInContainer defines MasterMacVlan plugin using linkInContainer feature.
+func (plugin *MasterMacVlanPlugin) WithLinkInContainer() *MasterMacVlanPlugin {
+	glog.V(100).Infof("Adding linkInContainer configuration to MasterMacVlanPlugin")
+
+	if plugin.masterPlugin == nil {
```
Comment (trewest) [inline pkg/nad/masterplugin.go:108]: Maybe safe to just also check if plugin == nil also

**Review by trewest — COMMENTED** [review]
Small comment

File: pkg/nad/masterplugin.go (hunk 1 above)
Comment (kononovn) [reply pkg/nad/masterplugin.go:108]: According to function NewMasterMacVlanPlugin this condition (plugin == nil) will be always false

**Comment by kononovn** [comment]
@trewest PTAL

**Review by trewest — APPROVED** [review]
/lgtm

**Pull request merged**